Carwash/users/admin.py

```diff
@@ -1,3 +1,31 @@
 from django.contrib import admin
 
-# Register your models here.
+from users.models import Users, Accounts
+
+
+@admin.register(Accounts)
+class AccountsAdmin(admin.ModelAdmin):
```

Comment:
I don't see any custom actions in the admin panel

Reply:
I made another commit with the needed changes.